src/plugins/intel_gpu/src/graph/impls/ocl/kernel_selector_helper.cpp

```diff
@@ -121,6 +121,47 @@ bool query_local_block_io_supported(engine& e, const ExecutionConfig& config) {
 
 namespace cldnn {
 
+    bool check_cm_jit_support(cldnn::engine& e, const cldnn::ExecutionConfig& config) {
```

Comment:
nit: please align the indentation with other code -- we don't use indentation for the content of namespace

Reply:
Done a875efa, dfafa15